app/repositories/ClientRepository.scala

```diff
@@ -33,7 +33,15 @@ class ClientRepository @Inject()(mongoComponent: MongoComponent)(implicit ec: Ex
 
 	def update(updatedClient: Client): Future[Boolean] = ???
 
-	def delete(crn: String): Future[Boolean] = ???
+	def delete(crn: String): Future[Boolean] = collection.find(equal("crn", crn)).toFuture().flatMap(client => {
```

Comment:
The find call is unnecessary, it was used in the other method to check the different outcomes of that call but here there is only delete or not deleted so we can simplify it by removing the find.